config-model/src/main/java/com/yahoo/vespa/model/container/ContainerClusterImpl.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.model.container;
+
+import com.yahoo.config.model.deploy.DeployState;
+import com.yahoo.config.model.producer.AbstractConfigProducer;
+
+/**
+ * The default container cluster implementation.
+ *
+ * @author gjoranv
+ */
+public final class ContainerClusterImpl
```

Comment:
Consider renaming this to `DefaultContainerCluster` or `ApplicationContainerCluster` or similar. _Impl_ suffix give little information on what the class does.